tgui/packages/tgui/interfaces/MODsuit.js

```diff
@@ -140,79 +141,88 @@ const HealthAnalyzer = (props, context) => {
     useroxy,
   } = props;
   return (
-    <>
-      <Section title="Health">
-        <ProgressBar
-          value={active ? userhealth / usermaxhealth : 0}
-          ranges={{
-            good: [0.5, Infinity],
-            average: [0.2, 0.5],
-            bad: [-Infinity, 0.2],
-          }}>
-          <AnimatedNumber value={active ? userhealth : 0} />
-        </ProgressBar>
-      </Section>
-      <Stack textAlign="center">
-        <Stack.Item grow>
-          <Section title="Brute">
-            <ProgressBar
-              value={active ? userbrute / usermaxhealth : 0}
-              ranges={{
-                good: [-Infinity, 0.2],
-                average: [0.2, 0.5],
-                bad: [0.5, Infinity],
-              }}>
-              <AnimatedNumber value={active ? userbrute : 0} />
-            </ProgressBar>
-          </Section>
-        </Stack.Item>
-        <Stack.Item grow>
-          <Section title="Burn">
-            <ProgressBar
-              value={active ? userburn / usermaxhealth : 0}
-              ranges={{
-                good: [-Infinity, 0.2],
-                average: [0.2, 0.5],
-                bad: [0.5, Infinity],
-              }}>
-              <AnimatedNumber value={active ? userburn : 0} />
-            </ProgressBar>
-          </Section>
-        </Stack.Item>
-        <Stack.Item grow>
-          <Section title="Toxin">
-            <ProgressBar
-              value={active ? usertoxin / usermaxhealth : 0}
-              ranges={{
-                good: [-Infinity, 0.2],
-                average: [0.2, 0.5],
-                bad: [0.5, Infinity],
-              }}>
-              <AnimatedNumber value={active ? usertoxin : 0} />
-            </ProgressBar>
-          </Section>
-        </Stack.Item>
-        <Stack.Item grow>
-          <Section title="Suffocation">
+    <Section>
+      {(!!show_vitals && (
```

Comment:
would prefer this be made into a ternary, so that the first parenthesis and double bang is not required

Reply:
out of town at the moment but i'll look into this when i can

Reply:
i have no idea if i did it right

Reply:
if only there were a tool for javascript that would let you know... 

(not your fault, i think writing the ui in TS is out of scope)